core/domain/suggestion_services_test.py

```diff
@@ -1345,6 +1351,83 @@ def test_get_reviewable_translation_suggestions(self):
         expected_language_code_list = ['hi', 'hi', 'pt']
         self.assertEqual(actual_language_code_list, expected_language_code_list)
 
+    def test_get_reviewable_translation_suggestions_with_valid_exp_ids( # pylint: disable=line-too-long
+            self):
+        # Add few translation suggestions in different languages.
+        self._create_translation_suggestion_with_language_code('hi')
+        self._create_translation_suggestion_with_language_code('hi')
+        self._create_translation_suggestion_with_language_code('pt')
+        self._create_translation_suggestion_with_language_code('bn')
+        self._create_translation_suggestion_with_language_code('bn')
+        # Add few question suggestions.
+        self._create_question_suggestion_with_skill_id('skill1')
+        self._create_question_suggestion_with_skill_id('skill2')
+        # Provide the user permission to review suggestions in particular
+        # languages.
+        user_services.allow_user_to_review_translation_in_language(
+            self.reviewer_id_1, 'hi')
+        user_services.allow_user_to_review_translation_in_language(
+            self.reviewer_id_1, 'pt')
+        # Get all reviewable translation suggestions.
```

Comment:
Prefer a new line above to differentiate between the test setup and the action under test. This comment is then probably not necessary. Here and below.

Reply:
Yep, just following what was there already previously in the code-base (no new line but just a comment). Now, I added a new line and also added a comment, to be explicit.